components/AudioPlayer.module.css

```diff
@@ -0,0 +1,198 @@
+.audio_player_section_container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  padding: 10%;
+}
+
+.image_wrapper {
+  position: relative;
+  top: 30px;
+  width: 180px;
+  height: 180px;
+  margin: 0 auto;
+  box-shadow: 0 0 20px black;
+}
+
+.episode_info_section{
+  position: relative;
+  top: 50px;
+  display: block;
+  text-align: center;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.episode_title {
+  text-align: center;
+}
+
+.episode_details{
+  width: 100%;
+  height: 150px;
```

Comment:
![スクリーンショット 2021-12-12 21 59 10](https://user-images.githubusercontent.com/51708229/145772859-285aa7ca-489a-42c7-bbb5-fdf6c58bfc2f.png)
@bluwakpa You mentioned this in the comment but it would have been better if you placed the screenshot like this.  It would make it easier to understand what you mean visually.

You specified `height: 150px`- again this is an absolute unit.  Because of this specification, the size of "episode_details" stays the same and won't correspond to the size of the screen. (= the height of the screen)

Also, the screenshot below is an example when the height of the screen is quite big and there is an unreasonable space under the "episode_details".
![スクリーンショット 2021-12-12 21 59 22](https://user-images.githubusercontent.com/51708229/145772868-7485ff5e-0ad4-429d-b670-584567c4abbe.png)

To prevent this, try to use `grid` to ".audio_player_section_container" and specify the size of each element.
Referring to `about.module.css` might be a hint for this styling.

If you need a session, please let me know.